jina/clients/base/http.py

```diff
@@ -19,6 +20,45 @@
 class HTTPBaseClient(BaseClient):
     """A MixIn for HTTP Client."""
 
+    def _graphql_mutate(
+        self,
+        mutation: str,
+        variables: dict = None,
+        timeout: float = None,
+        headers: dict = None,
+    ):
+        """Perform a GraphQL mutation
+
+        :param mutation: the GraphQL mutation as a single string.
+        :param variables: variables to be substituted in the mutation. Not needed if no variables are present in the mutation string.
+        :param timeout: HTTP request timeout
+        :param headers: HTTP headers
+        :return: dict containing the optional keys ``data`` and ``errors``, for response data and errors.
+        """
+        proto = 'https' if self.args.https else 'http'
+        graphql_url = f'{proto}://{self.args.host}:{self.args.port}/graphql'
+        endpoint = SgqlcHTTPEndpoint(graphql_url)
```

Comment:
Do import lazy, and make use of `ImportExtensions` to give proper message when dependency not found